sat/cli/bootsys/cabinet_power.py

```diff
@@ -193,6 +198,13 @@ def do_cabinets_power_on(args):
         LOGGER.error(f'Failed to resume discovery: {err}')
         raise SystemExit(1)
 
+    LOGGER.info('Recreating cronjobs which have failed to have been scheduled on time.')
```

Comment:
I realized that this is now probably always going to re-create the hms-discovery cronjob because it will not have been scheduled yet immediately after it's been resumed. I discussed with Jack, and he's going to see about putting it in the `on_retry_action`.